sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -510,4 +510,21 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 
 #IfColumn x12_partners x12_version
 ALTER TABLE `x12_partners` DROP COLUMN `x12_version`;
-#EndIf
\ No newline at end of file
+#EndIf
+
+#IfNotRow2D list_options list_id page_validation option_id add_edit_event#theform_prov
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES
+('page_validation', 'add_edit_event#theform_prov', '/interface/main/calendar/add_edit_event.php?prov=true', 170, '{}', 1);
+#EndIf
```

Comment:
This is odd since this is already in the codebase (guessing is related to the merge conflict and I can easily deal with this when integrate it into codebase)

Reply:
It was showing as part of conflict in git conflict resolution. I thought maybe was missed on prior commit after checked database.sql so I added from conflict resolution!